downloader/src/main/kotlin/Downloader.kt

```diff
@@ -151,7 +151,8 @@ class Downloader {
             targetDir.safeDeleteRecursively(force = true)
             targetDir.safeMkdirs()
 
-            exception.addSuppressed(e)
+            e.initCause(lastCause)
```

Comment:
This change affects the messages of OrtIssues coming from the `Downloader` and breaks `IssueResolutions`.
I think the effort it causes is not worth it and prefer to not do this change.

Reply:
Are you sure it *does* break IssueResolutions, or is there only a chance that they will? Because IIRC most IssueResolutionsuse regular expressions, so there is a good chance that they (most of them) will simply continue to work.

Anyway, we all know the current way of defining IssueResolutions is fragile, and this is yet one more reason to implement them differently. I'll file an issue about it.

Reply:
I would need to double check if they really break - but I don't have time for this right now.
I remember last time this exception was changed they did break.